boot/bootutil/include/bootutil/crypto/sha256.h

```diff
@@ -27,6 +28,10 @@
 
 #if defined(MCUBOOT_USE_MBED_TLS)
     #include <mbedtls/sha256.h>
+    #include <mbedtls/version.h>
+    #if MBEDTLS_VERSION_NUMBER == 0x03000000
```

Comment:
We probably want something like `>= 0x03000000` so that it continues to work after there is a new release.

Reply:
mbedtls-3.0.0 is the latest tage. Not sure whether it works on future versions.
But should probably work. Will update it.